---
layout: paper
title: "A domain decomposition strategy for alignment of multiple biological sequences on multiprocessor platforms"
nickname: 2009-01-01-a-domain-decomposition-strategy-for-alignment-of-multiple-biological-sequences-on-multiprocessor-platforms
authors: "Saeed, Fahad; Khokhar, Ashfaq; "
year: "2009"
journal: "Academic Press Journal of Parallel and Distributed Computing"
volume: 69
issue:
pages: 666-677
is_published: True
image: /assets/images/papers/biorxiv.png
projects: []
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid:
preprint: 
supplement:

# Links
doi: "10.1016/j.jpdc.2009.03.006"
pmid:

# Data and code
github: []
neurovault:
openneuro: []
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

# Abstract

Multiple Sequences Alignment (MSA) of biological sequences is a fundamental problem in computational biology due to its critical significance in wide ranging applications including haplotype reconstruction, sequence homology, phylogenetic analysis, and prediction of evolutionary origins. The MSA problem is considered NP-hard and known heuristics for the problem do not scale well with increasing numbers of sequences. On the other hand, with the advent of a new breed of fast sequencing techniques it is now possible to generate thousands of sequences very quickly. For rapid sequence analysis, it is therefore desirable to develop fast MSA algorithms that scale well with an increase in the dataset size. In this paper, we present a novel domain decomposition based technique to solve the MSA problem on multiprocessing platforms. The domain decomposition based technique, in addition to yielding better …
