---
layout: paper
title: "NGS‐Integrator: A Tool for Combining Information from Multiple Genome‐Wide NGS Data Tracks Using Minimum Bayes Factors"
nickname: 2024-04-16-bottenhorn-salo-diva
authors: "Jung, Hyun Jun; Wen, Bronte; Chen, Lihe; Saeed, Fahad; Knepper, Mark A; "
year: "2019"
journal: The Federation of American Societies for Experimental Biology
volume: 33
issue:
pages: 637.2-637.2
is_published: True
image: /assets/images/papers/acs.png
projects: []
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid:
preprint: 
supplement:

# Links
doi: "10.1096/fasebj.2019.33.1_supplement.637.2"
pmid:

# Data and code
github: []
neurovault:
openneuro: []
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

# Abstract

Genome‐wide studies that generate multiple high‐throughput next‐generation sequencing (NGS) datasets to identify genomic DNA elements for gene transcription require data integration methods to minimize complexity and false‐positive findings. This can involve integration of multiple genome‐wide data generated from same type or different types of high‐throughput NGS techniques. Since several strategies to integrate multiple genome‐wide NGS datasets based on peak calling tools have been developed, these conventional methods are typically applied to individual replicates and not the aggregate data from multiple data. NGS‐integrator, a Java‐based tool, integrates genome‐wide NGS datasets into a single data track for a genome browser based on minimum Bayes Factor (MBF) calculated from the signal‐to‐noise ratio. NGS‐integrator consists of two elements, “Calculator” and “Integrator”. The …
